resources/styles/components/reference-book/navbar.less

```diff
@@ -15,9 +24,26 @@
       }
     }
     .menu-toggle:before{ content: @fa-var-list; }
+    // don't collapse navbar when screen is narrow
+    @media (max-width: 979px) {
```

Comment:
Is there a var we can reference here? like `@book-content-collapse-breakpoint` ?